appium/webdriver/common/touch_action.py

```diff
@@ -24,16 +24,23 @@
 # pylint: disable=no-self-use
 
 import copy
+from typing import Dict, List, TypeVar
 
 from appium.webdriver.mobilecommand import MobileCommand as Command
 
+# from appium.webdriver.webelement import WebElement as WebElement
```

Comment:
Are these comments needed?

Reply:
No, it's just to avoid cyclic import to run unit test. I'll handle it later.
Same for others.